contracts/0.4.24/Lido.sol

```diff
@@ -430,14 +435,22 @@ contract Lido is StETHPermit, AragonApp {
     }
 
     /**
-     * @dev Sets limit on amount of ETH to withdraw from execution layer rewards vault per LidoOracle report
-     * @param _limitPoints limit in basis points to amount of ETH to withdraw per LidoOracle report
+     * @dev Set max positive rebase allowed per single oracle report
+     * token rebase happens on total supply adjustment,
```

Comment:
Still missing unit of measurement explainer. What 100% means? 100% of what? Instant APR? So we need a formula here. Or a small guide "how to choose right maxPositiveLimit"

Reply:
I hope it's fixed